src/Form/FormControl.js

```diff
@@ -9,7 +9,7 @@ import { isDirty } from '../Input/Input';
 
 export const styleSheet = createStyleSheet('MuiFormControl', theme => ({
   root: {
-    display: 'flex',
+    display: 'inline-flex',
```

Comment:
@oliviertassinari  Is this a bug that needs to be fixed or something that made mandatorily